plugin/EstateListHandle.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace onOffice\WPlugin;
+
+class EstateListHandle
+{
+
+	private $_listCustomField = array();
```

Comment:
Would it be possible to eliminate `$_listCustomField ` to make the class stateless?